backend/dataall/base/utils/naming_convention.py

```diff
@@ -4,10 +4,22 @@
 
 
 class NamingConventionPattern(Enum):
-    S3 = {'regex': '[^a-zA-Z0-9-]', 'separator': '-', 'max_length': 63}
+    S3 = {
+        'regex': '[^a-zA-Z0-9-]',
+        'separator': '-',
+        'max_length': 63,
+        'valid_external_regex': '^[a-z0-9][a-z0-9.-]{1,61}[' 'a-z0-9]$',
```

Comment:
should it match on trailing `' '` ?

Reply:
There is a thread with the discussion here
https://stackoverflow.com/questions/50480924/regex-for-s3-bucket-name/62673054#62673054

Reply:
From the above thread and other sites the regex I am seeing is `(?!(^xn--|.+-s3alias$))^[a-z0-9][a-z0-9-]{1,61}[a-z0-9]$` rather than the one above which also allows `.` and `' '`